data/json/items/gun/545x39.json

```
@@ -1,27 +1,39 @@
 [
   {
-    "id": "ak74",
-    "copy-from": "rifle_auto",
+    "id": "ak74_semi",
+    "copy-from": "rifle_semi",
     "looks_like": "modular_ar15",
     "type": "GUN",
-    "name": { "str": "AK-74M" },
-    "//": "AKs likewise aren't commercially traded in the US, plus this is newer.",
-    "description": "The successor to the well-known AK-47 rifle.  It combines the reliability of the AK series with the high-velocity, lightweight 5.45x39mm cartridge.",
+    "name": { "str": "5.45mm fighting rifle" },
```

```suggestion
    "name": { "str": "Russian assault rifle" },
```

I feel like this gets the point across better for those playing without brand names on. (Yes I'm aware this is a Soviet rifle, and that it's not actually an assault rifle if it's not automatic).